### PR TITLE
4029 remove xdd

### LINKED ISSUE #4029: Remove xdd

### PR DESCRIPTION
**Pull request by dvince2**
# Description

* Removing xdd as a document source.
* This will temporary break facets.

Resolves #4029

### CODE REVIEW

File: packages/server/src/main/resources/application-staging.properties
Comment (YohannParis) [inline packages/server/src/main/resources/application-staging.properties:1]: Why the clean up here?

File: packages/client/hmi-client/vite.config.local.ts
Comment (YohannParis) [inline packages/client/hmi-client/vite.config.local.ts:1]: Why the change to 8888?

File: packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue
Comment (YohannParis) [inline packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue:1]: This component should be deleted.

**Review by YohannParis — APPROVED** [review]

File: packages/server/src/main/resources/application-staging.properties
Comment (dvince2) [reply packages/server/src/main/resources/application-staging.properties:1]: Oops! accidental. this was from me using staging's keycloak for this. Ignore - reverting.

File: packages/client/hmi-client/vite.config.local.ts
Comment (dvince2) [reply packages/client/hmi-client/vite.config.local.ts:1]: Thank you. reverting

File: packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue
Comment (dvince2) [reply packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue:1]: This is still used by the model explorer for esgf data vs local epi data

File: packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
-			</DataTable>
-			<DataTable :value="relatedTerariumDatasets">
-				<Column field="name" header="Datasets" />
-			</DataTable>
-			<DataTable :value="relatedTerariumDocuments">
-				<Column field="name" header="Documents" />
-			</DataTable>
-		</AccordionTab>
-	</Accordion>
+	<section>
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue:2]: No need for this `<section>`

File: packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue [hunk 2]
```diff
@@ -2,15 +2,6 @@
...
-				<template v-slot:content>
-					<tera-facets-panel
-						v-if="viewType === ViewType.LIST"
-						:facets="facets"
-						:filtered-facets="filteredFacets"
-						:result-type="resourceType"
-						:docCount="docCount"
-					/>
-				</template>
 			</tera-slider-panel>
```
Comment (shawnyama) [inline packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue:5]: If this contains nothing now, should the `tera-slider-panel` be deleted or are there still plans to use it?

File: packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue
Comment (shawnyama) [inline packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue:1]: If this is still going to be used why were this components props and dropdown props removed? I don't think the dataset options are being passed through here now.

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue
Comment (dvince2) [reply packages/client/hmi-client/src/page/data-explorer/components/tera-filter-bar.vue:1]: Thanks for these... we're going to be doing more code cleanup on search, etc... ive made notes of these for there
